source/contribute/index.html.md.erb

```diff
@@ -22,5 +22,10 @@ GOV.UK Pay’s main application code is public and freely available under an MIT
 | [pay-publicapi](https://github.com/alphagov/pay-publicapi) | GOV.UK Pay public API endpoint. | Used by partner services to manage payments, generate reporting and configure their accounts. |
 | [pay-publicauth](https://github.com/alphagov/pay-publicauth) | GOV.UK Pay API authentication service. | Used by pay-publicapi to validate API tokens and by pay-selfservice to manage API tokens. |
 | [pay-selfservice](https://github.com/alphagov/pay-selfservice) | GOV.UK Pay self service application. | Used by authenticated users of our partner services to administer their accounts. |
+| [pay-products](https://github.com/alphagov/pay-products) | Products integrating with GOV.UK Pay in Java(Dropwizard). | Manages products that a Government Service would like to take payments for. These products are integrated with GOV.UK Pay so that a user can make payments using a provided URL. |
+| [pay-products-ui](https://github.com/alphagov/pay-products-ui) | User interface app for product payments integrated with GOVUK Pay. | TBC. |
```

Comment:
High level overview of some of the `TBC` uses, again I think this could do with some group opinion on the language we use now vs. when the repos were created. 

(trying to roughly follow the pattern used in the rest of the table)

Products UI - Used to start payment journeys for payment links, optionally allowing users to enter payment amount and reference.

Direct Debit Connector - Used to interface with Direct Debit providers.

Direct Debit frontend - Used to collect Direct Debit information from our users.

Ledger - Used to store all historic payments and refunds after they have been processed by Card Connector.